### PR TITLE
adding a layered calorimeter struct that copes with conical calorimeters

### PR DESCRIPTION
**Pull request by Voutsi**
BEGINRELEASENOTES
- adding a layered calorimeter struct that copes with conical calorimeters

ENDRELEASENOTES

### CODE REVIEW

File: DDRec/include/DDRec/DetectorData.h [hunk 1]
```diff
@@ -449,6 +449,150 @@ namespace DD4hep {
...
+
+
+
+    /** Simple data structure defining a layered calorimeter layout for
+     *  reconstruction. Modified version of the above LayeredCalorimeterStruct that
+     *  can deal with a conical shaped calorimeter
+     * 
+     * @author Y.Voutsinas, CERN
+     * @date March, 2017
+     * @version $Id: $
```
Comment (petricm) [inline DDRec/include/DDRec/DetectorData.h]: $Is is a SVN tag

**Comment by gaede** [comment]
Could we not simply have:
```c++
      /// enum for encoding the sensor type in typeFlags
      enum LayoutType{
	BarrelLayout=0,
	EndcapLayout,
        ConicalLayout
      };

      /// type of layout: BarrelLayout, EndcapLayout or ConicalLayout
      LayoutType layoutType ;

      /** extent of the calorimeter in the r-z-plane [ rmin, rmax, zmin, zmax, rmin2, rmax2 ] in mm.
         * where rmin2, rmax2 are the radii at zmax for the ConicalLayout and not used else.
         */
      double extent[6] ;

```

**Comment by petricm** [comment]
I agree, and then you can use `ConicalLayout` in the overloading of `<<`

**Comment by Voutsi** [comment]
Thanks for your suggestions, I modified accordingly.

**Comment by petricm** [comment]
@Voutsi please modify(edit comment) in the initial comment in the release notes tag to reflect the actual change.

**Comment by Voutsi** [comment]
release notes updated

**Pull request merged**